core/osrd-reporting/src/main/java/fr/sncf/osrd/reporting/exceptions/OSRDError.java

```diff
@@ -1,10 +1,12 @@
 package fr.sncf.osrd.reporting.exceptions;
 
-import com.squareup.moshi.*;
 import com.squareup.moshi.JsonAdapter;
+import com.squareup.moshi.KotlinJsonAdapterFactory;
```

Comment:
```suggestion
import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory;
```

There's one of the two versions that's deprecated